javaagent-extension-api/src/main/java/io/opentelemetry/javaagent/bootstrap/internal/CommonConfigSetter.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.bootstrap.internal;
+
+import java.util.function.Consumer;
+import java.util.function.Supplier;
+
+/**
+ * This class is internal and is hence not for public use. Its APIs are unstable and can change at
+ * any time.
+ */
+public final class CommonConfigSetter {
+  private CommonConfigSetter() {}
+
+  public static <T> void set(Supplier<T> supplier, Consumer<T> consumer) {
```

Comment:
move this method into HttpClientInstrumenterFactory and make private?

Reply:
it will also be used for HttpServerInstrumenterFactory later

Reply:
changed